terraform/modules/proxmox/cloudinit/bootloader.sh

```
@@ -17,15 +17,15 @@ function error() {
 }
 
 function error_with_log() {
-  curl -X POST                                                                                                                                                \
-       -H "Content-Type: application/json"                                                                                                                    \
-       -d "{\"chat_id\": \"${telegram_chat_id}\", \"text\": \"${domain}: ${1}\nReason:\\n\", \"disable_notification\": false}" \
+  curl -X POST                                                                                                                      \
+       -H "Content-Type: application/json"                                                                                          \
+       -d "{\"chat_id\": \"${telegram_chat_id}\", \"text\": \"${domain}: ${1}\nReason:\\n\", \"disable_notification\": false}"      \
         https://api.telegram.org/bot${telegram_bot_token}/sendMessage
   curl -v -F "chat_id=${telegram_chat_id}" -F document=@$2 https://api.telegram.org/bot${telegram_bot_token}/sendDocument
 	exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)